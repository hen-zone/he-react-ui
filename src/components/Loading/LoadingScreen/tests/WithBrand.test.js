import createTestContext from 'react-cosmos-test/enzyme'
import fixture from '../fixtures/WithBrand.fixture'

const { mount, getWrapper } = createTestContext({ fixture })

beforeEach(mount)

test('<LoadingScreen /> rendered correctly with branded fixture', () =>
  expect(getWrapper()).toMatchSnapshot())
