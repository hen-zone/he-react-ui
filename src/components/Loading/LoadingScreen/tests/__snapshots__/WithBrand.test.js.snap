// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingScreen /> rendered correctly with branded fixture 1`] = `
<LoadingScreen>
  <div
    className="loadingContainer"
  >
    <div
      className="pulseButton"
    />
    <div
      className="pulseButton2"
    >
      <HealthEngine
        className="loadingIcon"
      >
        <svg
          className="loadingIcon"
          version="1.1"
          viewBox="0 0 48 38"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M12.9079345,32.7452837 C13.1763914,35.6739197 15.6350174,37.9154191 18.5788836,37.9154191 C21.5227499,37.9154191 23.9813759,35.6739197 24.2498328,32.7452837 L24.2498328,25.1600585 L12.9079345,25.1600585 L12.9079345,32.7452837 L12.9079345,32.7452837 Z"
            fill="#5BC0C9"
            id="Shape"
          />
          <path
            d="M24.2498328,6.2159583 L24.2498328,6.2159583 C24.4467061,4.06823902 23.4109333,1.99397789 21.575,0.8592795 C19.7390667,-0.275418886 17.4187006,-0.275418886 15.5827673,0.8592795 C13.7468339,1.99397789 12.7110612,4.06823902 12.9079345,6.2159583 L12.9079345,6.2159583 L12.9079345,13.8011836 L24.2498328,13.8011836 L24.2498328,6.2159583 Z"
            fill="#5BC0C9"
            id="Shape"
          />
          <path
            d="M37.1177539,25.1410954 L31.4230769,25.1410954 L31.4230769,32.7263207 C31.6915338,35.6549566 34.1501598,37.896456 37.0940261,37.896456 C40.0378923,37.896456 42.4965183,35.6549566 42.7649752,32.7263207 L42.7649752,19.4521764 C42.7650838,22.5756585 40.2442888,25.1150664 37.1177539,25.1410954 L37.1177539,25.1410954 Z"
            fill="#C6E400"
            id="Shape"
          />
          <path
            d="M37.1177539,0.546002402 C33.9726707,0.546002402 31.4230769,3.09301818 31.4230769,6.23492137 L31.4230769,13.8201467 L37.1177539,13.8201467 C40.2628372,13.8201467 42.8124309,16.3671624 42.8124309,19.5090656 L42.8124309,6.23492137 C42.8124309,3.09301818 40.2628372,0.546002402 37.1177539,0.546002402 L37.1177539,0.546002402 Z"
            fill="#C6E400"
            id="Shape"
          />
          <path
            d="M5.29604959,13.8201467 C2.36444943,14.0883321 0.120681325,16.5444721 0.120681325,19.4853618 C0.120681325,22.4262514 2.36444943,24.8823915 5.29604959,25.1505769 L12.8889522,25.1505769 L12.8889522,13.8201467 L5.29604959,13.8201467 L5.29604959,13.8201467 Z"
            fill="#2B4B5A"
            id="Shape"
          />
          <rect
            fill="#2B4B5A"
            height="11.3304303"
            id="Rectangle-path"
            width="7.2"
            x="24.2498328"
            y="13.8201467"
          />
          <rect
            fill="#008EC8"
            height="11.3304303"
            id="Rectangle-path"
            width="11.3418983"
            x="12.9079345"
            y="13.8201467"
          />
          <path
            d="M37.5279213,13.8201467 L31.4332443,13.8201467 L31.4332443,25.1410954 L37.9279213,25.1410954 C40.8381324,24.8492731 42.6538429,22.4024989 42.8538429,19.480621 C42.8538429,16.5587431 40.8381324,14.111969 37.9279213,13.8201467 L37.5279213,13.8201467 Z"
            fill="#48B400"
            id="Shape"
          />
        </svg>
      </HealthEngine>
    </div>
  </div>
</LoadingScreen>
`;
